app/controllers/va_facilities_controller.rb

```diff
@@ -1,30 +1,36 @@
 class VaFacilitiesController < ApplicationController
-  include PracticeUtils
+  include PracticeUtils, VaFacilitiesHelper
   before_action :set_va_facility, only: [:show, :created_practices, :update_practices_adopted_at_facility]
-
   def index
-    @facilities = VaFacility.cached_va_facilities.select(:common_name, :id, :official_station_name).order_by_station_name
+    @facilities = VaFacility.cached_va_facilities.select(:common_name, :id, :visn_id, :official_station_name).order_by_station_name.includes([:visn])
+    @clinical_resource_hubs = ClinicalResourceHub.cached_clinical_resource_hubs
+    @facilities = (@facilities.includes(:visn).sort_by(&:official_station_name.downcase) + @clinical_resource_hubs.includes([:visn]).sort_by(&:id))
```

Comment:
Can we rename the first `@facilities` variable to something else. We're overwriting it, so it doesn't make sense to make it the same name. Maybe we can do something like `va_facilities`? Also, since it doesn't look like we need it in the view, we can remove the `@` and just make it a regular variable.

Reply:
Got it - renamed @clinical_resource_hubs to just clinical_resource_hubs as well.